README.md

```diff
@@ -242,6 +243,43 @@ Methods and variables that are intended to be "private" should begin with a lead
 _privateMethod: ->
 ```
 
+<a name="objects"></a>
+## Objects
+
+When declaring object literals:
+
+- With one key, value pair use a single line
+
+```coffeescript
+car = make: "Toyota"
```

Comment:
Why use double quotes here instead of single? I'd vote for uniform single across the board, except with string interpolation